src/SDKs/_metadata/maintenance_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```

Comment:
This file is in the wrong path, it should be in `eng\mgmt\mgmtmetadata\`. Could you run

> msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules

then re-run `generate.ps1`

Reply:
Looks like my repo fork was done when all RPs had Caps folder names, and the eng\mgmt.proj did not exist. I did a merge from master repo and will update.

Reply:
Could you remove this file then?